### ISSUE TITLE
При загрузке файлов в приложение $mol_attach непонятно что загружено

### ISSUE THREAD

**Issue by NatalyaBudnikova** [eigenmethod/mol#205]
**Приложение:** [$mol_attach](http://mol.js.org/#filter/gist=180442153/demo=mol_attach_demo/supply=75BCD15)
**Браузеры:**  
        Chrome 58.0.3029.81 (64-bit)
        Safari, iOS 9.3.5 (iPad)
        Microsoft Edge 20.10240.16384.0 (Windows 10, IRBIS TW38)
        Chrome 57.0.2987.132, Android 6.0.1 (Samsung)
**Детали:**

_Шаги воспроизведения:_
1. Запустить приложение $mol_attach
2. Загрузить файлы (не картинки)

_Ожидаемое поведение_
Визуально можно понять какой файл был загружен (название, расширение).
Еще есть вопрос: изменяется размеры прямоугольников (файлов) в зависимости от того сколько файлов загружено, ожидаемо ли это? или они должны быть фиксированного размера?

_Текущее поведение_
Не понятно какие файлы были загружены
![_001](https://cloud.githubusercontent.com/assets/25827903/25609379/af5fd1f8-2f27-11e7-9006-453a49e12b4d.png)

**Comment by nin-jin** [comment]
Да, размеры намеренно подстраиваются так, чтобы оптимально заполнять пространство.

**Comment by nin-jin** [comment]
Сейчас этот компонент не очень полноценен. Думаю нужна его полная переработка.

**Comment by PavelZubkov** [comment]
Что должно быть в полноценном?

У меня пока только одна проблема возникала - передлать его на одиночный аттач файла, вместо массового

По поводу понятности, что за файл:
- индикация типа: если не картинка доставть из расширения/mime и показывать текст вместо картинки
- нужен способ видеть имя файла(загрузил что-то лишнее и не понятно что удалять)

**Comment by nin-jin** [comment]
У полноценного должны быть кнопочки удаления как минимум.
